lib/services/access_monitoring_rules.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package services
+
+import (
+	"context"
+
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport/api/client/accessmonitoringrules"
+	"github.com/gravitational/teleport/api/types/accessmonitoringrule"
+	"github.com/gravitational/teleport/lib/utils"
+)
+
+var _ AccessMonitoringRules = (*accessmonitoringrules.Client)(nil)
+
+// AccessMonitoringRules is the AccessMonitoringRule service
+type AccessMonitoringRules interface {
+	CreateAccessMonitoringRule(ctx context.Context, in *accessmonitoringrule.AccessMonitoringRule) (*accessmonitoringrule.AccessMonitoringRule, error)
```

Comment:
Do we really need Create, Update and Upsert API methods?

Reply:
Yes but Upsert may not be required to be exposed to the gRPC service is specifically what I'm saying

Reply:
Oh missed that you mentioned that, my bad